pkg/util/constants/constants.go

```diff
@@ -28,4 +28,8 @@ const (
 	PVProvisionerGCEPD  = "kubernetes.io/gce-pd"
 	PVProvisionerAWSEBS = "kubernetes.io/aws-ebs"
 	PVProvisionerNone   = "none"
+
+	EtcdCliCertFile = "etcd-crt.pem"
```

Comment:
we probably should stop adding stuff into this global const pkg. we should move the const back to where they belong to.